finish/src/main/java/io/openliberty/guides/inventory/InventoryResource.java

```diff
@@ -41,27 +41,40 @@
     @GET
```

Comment:
This file needs a copyright update

Reply:
Updated.

Reply:
Remove 2018, should just read as `2017, 2019`